vars/fedora-32.yml

```diff
@@ -15,4 +15,4 @@
 # limitations under the License.
 
 _yum:
-  - { state: "latest", name: "firefox" }
+  - { state: "latest", name: "https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm" }
```

Comment:
Please do in this way:
- Run molecule so start the VM but don't install any RPM
- `molecule login` into the VM and follow official installation guide to install the RPM
- Check the result `/etc/yum.repo.d/???`, see how official RPM populate the repo file
- Now implement the `_rpm_key` and `_yum_repository` here
- Now we could install the package with package name here, simply ;-(